pkg/proxy/nftables/proxier.go

```diff
@@ -63,6 +64,16 @@ const (
 	// so they don't need any "kube-" or "kube-proxy-" prefix of their own.
 	kubeProxyTable = "kube-proxy"
 
+	// base chains
+	kubeFilterPreRouting     = "filter-prerouting"
```

Comment:
"Prerouting" (and "Postrouting"); be consistent between the two forms. If it was `filter-pre-routing` then `FilterPreRouting` would be right, but it's not.